packages/@aws-cdk/assertions/lib/template.ts

```diff
@@ -24,7 +24,7 @@ export class Template {
    * nested set of records.
    * @param template the CloudFormation template formatted as a nested set of records
    */
-  public static fromTemplate(template: { [key: string] : any }): Template {
+  public static fromObject(template: { [key: string] : any }): Template {
```

Comment:
fromJSON()

Reply:
?

Reply:
see comment on here - https://github.com/aws/aws-cdk/pull/15857#discussion_r681601653

still not sure this should change.